src/api/imports.ts

```diff
@@ -138,7 +163,8 @@ function createImportsApi({
             workerDone = true
           })
 
-        function handleFirstProgressMessage(message: PortMessage) {
+        function handleFirstProgressMessage(message: unknown) {
+          assertIsPortMessage(message)
```

Comment:
Not strictly related, but I added this assertion function and figured it could be useful here too.